auth/password_check.js

```diff
@@ -0,0 +1,8 @@
+function password_check(){
```

Comment:
Why would you call this password check?
Why not revealPassword?

You are still mixing camel case with snake case here

Reply:
Also this method can be a toggle the if is not necessary
Something like get_password.type = !get_password.type

Reply:
sijamix the naming ....nimetumia snake case all through hapo